### PR TITLE
feat(connect): add support for StellarClaimClaimableBalanceOp.

### PR DESCRIPTION
**Pull request by mroz22**
new messages for ChangeLanguage feature, will be implemented in #11183
new messages for Stellar. @overcat is looking into it 🥇 

edit: aha, I can see that stellar tests are probably written in that way that they require implementation of the new message https://github.com/trezor/trezor-suite/actions/runs/7927610555/job/21644408103?pr=11210#step:7:749

### CODE REVIEW

File: packages/protobuf/messages.json [hunk 1]
```diff
@@ -4989,6 +5031,13 @@
                 "firmware_header": {
                     "type": "bytes",
                     "id": 2
+                },
+                "language_data_length": {
+                    "type": "uint32",
+                    "id": 3,
+                    "options": {
+                        "default": 0
+                    }
```
Comment (mroz22) [inline packages/protobuf/messages.json:5040]: what is this field @Hannsek btw?

Comment (Hannsek) [reply packages/protobuf/messages.json:5040]: 🤷🏻‍♂️ @grdddj @matejcik

**Comment by overcat** [comment]
The changes related to Stellar are added in #11213

File: packages/protobuf/messages.json (hunk 1 above)
Comment (matejcik) [reply packages/protobuf/messages.json:5040]: length of language blob update. we'll tell you on Monday

**Comment by mroz22** [comment]
any volunteer to approve this? I already approved [5a908b7](https://github.com/trezor/trezor-suite/pull/11210/commits/5a908b743c6a0330b070843d0f402ab9ecb89131) and [8c17691](https://github.com/trezor/trezor-suite/pull/11210/commits/8c176918a0f1dc1bbfd1411e95133dc8abf73d1d) but since I created this PR I can't mergre myself.

what about @martykan

**Review by martykan — APPROVED** [review]

**Pull request merged**